helm/prometheus-meta-operator/values.yaml

```diff
@@ -26,6 +26,31 @@ alertmanager:
     apiURL: ""
   storage:
     size: 1Gi
+  teams:
+  - name: atlas
+    oncall: true
+    stableNotifyOnly: true
+    extraMatchers:
+    - name: type
+      matchType: "!="
+      value: "heartbeat"
+    - name: alertname
+      matchType: "!~"
+      value: "Inhibition.*"
```

Comment:
Feels weird that Atlas only has those extra matchers and not other teams, I guess no one want to receive heartbeat or inhibition.

Reply:
exactly

Reply:
Does it mean when a new team has routing alerts, we will have to release pmo ?

Reply:
Yes it's how it's done today